### ISSUE TITLE
Cannot navigate dropdown menus when  voiceover is on

### ISSUE THREAD

**Issue by MaryS94** [RITAccess/blockly#48]
Update: Dropdown menus can be navigated and read aloud however, the pop-up window is hidden and only the field itself changes. This happens because under the hood, the selection is being changed and not the highlight, when the selection is changed the pop-up menu is auto closed. A potential work around is overwriting the field select method to disable auto closing and instead only close when enter is pressed.

Old:Voiceover uses the arrow keys by default so dropdowns should be controlled with WASD to be consistent.

**Comment by MaryS94** [comment]
This now works with WASD

**Comment by retrogamer80s** [comment]
Great!

Steph
